app/views/search/_spatial_manual_entry.html.erb

```diff
@@ -0,0 +1,16 @@
+<div id="manual-coord-entry-container" class="manual-coord-entry" data-bind="with: query.spatialCondition">
+  <p data-bind="visible: !visible()"><i class="fa fa-fw fa-crop"></i></p>
+  <p data-bind="visible: visible()">
+    <label for="manual-coord-entry"><!-- ko text: spatialType --><!-- /ko -->: </label>
+    <!-- ko if: spatialType() == 'Point' -->
+    <input type="text" id="manual-coord-entry" name="manual-coord-entry" data-bind="value: coordinates, valueUpdate: 'blur'" placeholder="lon, lat (e.g. 130, 44.2)">
```

Comment:
It seems like it could simplify things a lot if you had separate inputs for lat and lon as well as separate inputs for south, west, north, and east in the bounding box case.